### DISCUSSION TITLE
Page offset not working when converting jpg to pdf on macOS

### DISCUSSION THREAD

**Discussion by brzGatsu** [ImageMagick/ImageMagick#6678] [Contribute]
### ImageMagick version

7.1.0-45

### Operating system

MacOS

### Operating system, version and so on

12.5

### Description

Hello, I use the below command on windows to position an arbitrary jpg at 260x340 at the middle center (offset towards the top) of an A2 sized PDF. When using the same command on macOS I get a blank page. I assume that the page offset and maybe the gravity command are not working. Maybe its related to my Retina display?

### Steps to Reproduce

1. Install imagemagick via brew
2. Use the following command (insert actual image path in the brackets)

```convert <imagepath>.jpg -resize 260x340 -gravity Center -extent 260x340 -density 96 -page a2+115.55+89.55 <imagepath>.pdf```
3. Notice blank PDF when in fact the image should be (almost) in the middle center.

**Comment by fmw42** [comment]
It is not blank for me. I am on Mac OSX Monterey and IM 6.9.12.65 Q16 using Ghostscript 9.55.

Perhaps it is your version of Imagemagick or Ghostscript.  What is your Ghostscript version?

**Comment by brzGatsu** [comment]
Hi, my GS version is 9.56.1.

**Comment by dlemstra** [comment]
Seems to work on IM6 but is broken on IM7 not yet sure why.
